x-pack/platform/plugins/shared/maps/public/connected_components/mb_map/tooltip_control/__snapshots__/tooltip_popover.test.tsx.snap

```diff
@@ -16,7 +16,7 @@ exports[`TooltipPopover render should render tooltip popover 1`] = `
   }
   closePopover={[Function]}
   display="inline-block"
-  hasArrow={true}
+  hasArrow={false}
```

Comment:
I think the arrow is very important to the map tooltip. Can you add `hasArrow={true}` to x-pack/platform/plugins/shared/maps/public/connected_components/mb_map/tooltip_control/tooltip_popover.tsx line 153

Without the arrow, its hard to tell the feature the tooltip is showing
<img width="377" height="233" alt="Screenshot 2025-11-26 at 9 30 48 AM" src="https://github.com/user-attachments/assets/bf33c066-2975-44d1-afbd-e5661faf09a4" />

With the arrow, its much easier to see which feature the tooltip is showing
<img width="337" height="325" alt="Screenshot 2025-11-26 at 9 30 31 AM" src="https://github.com/user-attachments/assets/61a9da5b-fadf-42e7-9960-a37b6f244731" />

Reply:
<img width="711" height="538" alt="image" src="https://github.com/user-attachments/assets/e479bf48-f1ff-4191-a149-49dbd6a715a8" />

Both popovers for tooltip and layer action are back